ui/app/AppLayouts/Profile/views/wallet/NetworksView.qml

```diff
@@ -100,10 +119,10 @@ Item {
                 StatusSwitch {
                     id: testnetSwitch
                     objectName: "testnetModeSwitch"
-                    checked: walletStore.areTestNetworksEnabled
-                    onToggled: {
-                        checked = Qt.binding(() => walletStore.areTestNetworksEnabled)
-                        Global.openTestnetPopup()
+                    checked: root.areTestNetworksEnabled
+                    onToggled:{
+                        checked = Qt.binding(() => root.areTestNetworksEnabled)
```

Comment:
Hmm, you shouldn't be setting the `checked` property from the `toggled` handler :thinking:

Reply:
hmm the binding should break when the user clicks the button, and we're not sure which value `root.areTestNetworksEnabled` will end up having (since we open a popup letting the user proceed or cancel the toggle. Where would you suggest I put this logic instead?

I guess what we really want is a "Detached Switch" that does not actually change its `checked` state when the user clicks it (it just emits some event), and has its state set externally instead.